app/views/portfolios/show.html.erb

```diff
@@ -1,12 +1,14 @@
-<h1>Portfolio</h1>
-<p>
-  <strong>Name: </strong><%= @portfolio.name %>
-</p>
-<p>
-  <strong>Asset manager: </strong><%= @portfolio.asset_manager.first_name %>
-</p>
-
-<h3>Buildings</h3>
-<% @portfolio.buildings.each do |b| %>
-  <p><%= b.name %> | <%= link_to('Details', building_path(b)) %></p>
-<% end %>
+<h1>Portfolio</h1>
+<p>
+  <strong>Name: </strong><%= @portfolio.name %>
+</p>
+<p>
+  <strong>Asset manager: </strong><%= @portfolio.asset_manager.first_name %>
+</p>
+
+<h3>Buildings</h3>
+<% @portfolio.buildings.each do |b| %>
+  <p><%= b.name %> | <%= link_to('Details', building_path(b)) %></p>
+<% end %>
+
+<%= javascript_pack_tag 'containers\\AssetManagerContainer.js' %>

```

Comment:
``` ruby
<%= javascript_pack_tag 'containers/AssetManagerContainer.js' %>
```
or else i get the error "can't find containers\AssetManagerContainer.js" because you used backslash instead of forward slash